index.html

```diff
@@ -0,0 +1,248 @@
+          </div>
+          <div class="program-description">
+            <p>
+              Ndi Umunyarwanda and Our Identity Journey, Rwanda Works! The Land
+              of a million possibilities and opportunities
+            </p>
+          </div>
+        </div>
+
+        <div class="program-bar">
+          <div class="program-icon">
+            <img src="./general-assets/refresh.png" alt="" />
+          </div>
+          <div class="program-title">
+            <span> Update </span>
+          </div>
+          <div class="program-description">
+            Participants discussed Rwanda’s socio-economic progress and proposed
+            solutions to current and future challenges.
+          </div>
+        </div>
+
+        <div class="program-bar">
+          <div class="program-icon">
+            <img src="./general-assets/confetti.png" alt="" />
+          </div>
+          <div class="program-title">
+            <span> Party </span>
+          </div>
+          <div class="program-description">
+            <p>
+              This edition had a special feature called the “Career Corner”;
+            </p>
+          </div>
+        </div>
+        <div class="program-bar">
+          <div class="program-icon">
+            <img src="./general-assets/dinner.png" alt="" />
+          </div>
+          <div class="program-title">
+            <span> Dinner </span>
+          </div>
+          <div class="program-description">
+            <p>
+              Over the last editions, Rwanda Day has brought together more than
+              35,000 Rwandans and Friends of Rwanda living in Europe,.
+            </p>
+          </div>
+        </div>
+      </div>
+      <button type="submit" name="Submit" class="join-button">
+        Join Rwanda Day 2021
+      </button>
+    </div>
+    <div class="featured-speakers">
+      <div class="speakers-title"><span> Featured speakers </span></div>
+      <div class="speakers-line"></div>
+      <div class="speaker-section"></div>
+    </div>
+    <div class="more">
+      <div class="more-button">MORE</div>
+    </div>
+    <section class="partners">
+      <div class="partners-title">
+        <h2>Partners</h2>
+      </div>
+      <div class="partners-logo">
+        <div class="partner-image">
+          <img src="general-assets/partner1.png" alt="" />
+        </div>
+        <div class="partner-image">
+          <img src="general-assets/partner2.png" alt="" />
+        </div>
+        <div class="partner-image">
+          <img src="general-assets/partner3.png" alt="" />
+        </div>
+        <div class="partner-image">
+          <img src="general-assets/partner4.png" alt="" />
+        </div>
+        <div class="partner-image">
+          <img src="general-assets/partner5.png" alt="" />
+        </div>
+      </div>
+    </section>
+
+    <div class="footer">
+      <div class="footer-logo">
+        <img src="desktop-assets/logo.jpg" alt="footer logo" />
```

Comment:
[OPTIONAL]

- [ ] Please take into consideration a reduction of the size of the logo in the footer.